client/src/components/PortfolioGraph.tsx

```diff
@@ -238,6 +239,15 @@ const PortfolioGraphWithoutI18: React.FC<PortfolioGraphProps> = ({ graphJSON, st
           },
         ]}
       />
+      <div className="warning-banner">
+        <div className="float-left">
+          <img src={warning} className="icon" alt="Warning" />
+          <span className="warning">
+            {/* eslint-disable-next-line */}
+            This diagram may show a network of portfolios. <a href="">Learn more.</a>
```

Comment:
Mind wrapping this in a `<Trans>` tag and running `yarn lingui extract --clean` in the `/client` directory to internationalize this snippet of text?

Reply:
Oh right, thanks for the reminder!